### DISCUSSION TITLE
Problem loading stat widgets onto page

### DISCUSSION THREAD

**Discussion by Nonentity5565** [anuraghazra/github-readme-stats#3271] [Q&A] [answered]
I deployed my own vercel instance and I noticed that this problem occurs pretty frequently when I load into my profile. The problem occurred on the GitHub mobile application as well so I'm certain it's not a web extension that's causing the error:
![image](https://github.com/anuraghazra/github-readme-stats/assets/8046214/cf98deb2-29de-40b4-a8dd-6432f25e0051)

The problem doesn't fix itself and requires a page refresh to load the stat widgets properly. 
![image](https://github.com/anuraghazra/github-readme-stats/assets/8046214/2a116bd2-3309-48ee-8ddc-bda91492db14)

Is this an issue on my vercel instance and do I need to change a config on vercel?

**Comment by Nonentity5565** [comment]
To save some time here is the code snippet of my widgets:
```
### My GitHub Stats:

<a href="https://github.com/anuraghazra/github-readme-stats">
  <img height=200 align="center" src="https://github-readme-stats-nonentity5565.vercel.app/api?username=Nonentity5565&show_icons=true&show_all_commits=true" />
</a>
<a href="https://github.com/anuraghazra/convoychat">
  <img height=200 align="center" src="https://github-readme-stats-nonentity5565.vercel.app/api/top-langs/?username=anuraghazra&layout=compact&langs_count=8&card_width=320" />
</a>
```

**Comment by qwerty541** [comment, marked as answer]
Hey, @Nonentity5565! Tih issue caused by timeout of GitHub CDN when it reuploads image from your instance. See https://github.com/anuraghazra/github-readme-stats/issues/2603 for more info. In future it can be resolved by migration on GitHub actions. You can track progress on this feature at https://github.com/anuraghazra/github-readme-stats/issues/2179.

**Reply by Nonentity5565** [reply]
Got it, thanks for the help 😄
